src/main/webapp/index.html

```diff
@@ -47,14 +51,18 @@
             services: {
                 identify: Libp2PIdentify.identify(),
                 ping: Libp2PPing.ping(),
-                dht: Libp2PKadDht.kadDHT({protocol: "dot/dht/1.0.0"}),
+                dht: Libp2PKadDht.kadDHT({protocol: "/dot/kad"}),
                 pubsub: ChainsafeLibp2PGossipsub.gossipsub(),
             }
         };
 
         libp = await Libp2P.createLibp2p(test);
         libp.start();
 
+        // libp.addEventListener('peer:discovery', (evt) => console.log('Discovered:', evt.detail.id.toString()))
+        // libp.addEventListener('peer:connect', (evt) => console.log('Connected:', evt.detail.toString()))
+        // libp.addEventListener('peer:disconnect', (evt) => console.log('Disconnected:', evt.detail.toString()))
```

Comment:
intentionally left commented out?

Reply:
Yes, the commented code in main.js is for testing purposes while developing the different protocols, and will be removed after all of them are done